dev/com.ibm.ws.logging_2_fat/publish/servers/fatTestPorts.xml

```diff
@@ -0,0 +1,19 @@
+<!--
```

Comment:
We don't need to explicitly add this file. This file is already added in automatically, as part of our FAT infrastructure. https://github.com/OpenLiberty/open-liberty/blob/integration/dev/fattest.simplicity/autoFVT-defaults/fatTestPorts.xml